src/App.js

```diff
@@ -115,34 +116,26 @@ class App extends React.Component {
      * * * * * * * * * * * * * */
 
     render() {
+        const { plot, ...rest } = this.state
+        const routeProps = {
```

Comment:
Could there be a better way to structure this than pushing about 15 props on route? 

Maybe using [contexts](https://reactjs.org/docs/context.html) ?

Reply:
Yes, I actually a branch where I've built the Provider pattern, so that'll come up next. 

I just needed to understand correctly which pieces of state are running through the App. For instance in that branch I have it divided as hexapod params, plot data, and the rest of settings. And that results in 3 context providers.